File: 백준/Bronze/1934. 최소공배수/최소공배수.py
import sys

def gcd(a, b):
    while b:
        a, b = b, a % b
    return a

def lcm(a, b):
    return (a * b) // gcd(a, b)

n = int(sys.stdin.readline())
results = []
for _ in range(n):
    a, b = map(int, sys.stdin.readline().split())
    results.append(lcm(a, b))

    
for result in results:
    print(result)

File: 백준/Bronze/1934. 최소공배수/최소공배수.js
const fs = require("fs");
const input = fs.readFileSync("/dev/stdin").toString().trim().split("\n");

function gcd(a, b) {
  while (b) {
    [a, b] = [b, a % b];
  }
  return a;
}

function lcm(a, b) {
  return Math.floor((a * b) / gcd(a, b));
}

const n = parseInt(input[0]);
const results = [];

for (let i = 1; i <= n; i++) {
  const [a, b] = input[i].split(" ").map(Number);
  results.push(lcm(a, b));
}

console.log(results.join("\n"));
